src/components/ConnectorName.tsx

```diff
@@ -1,21 +1,53 @@
+import QuestionMarkIcon from '@mui/icons-material/QuestionMark';
 import { Stack } from '@mui/material';
 import useConstant from 'use-constant';
 import { getConnectorName } from 'utils/misc-utils';
 
 interface Props {
     size?: number;
-    path: string;
+    path?: string;
```

Comment:
Making path optional so we don't have a bunch of hardcoded empty strings laying around.